configuration-service/handlers/service_resource.go

```diff
@@ -175,6 +175,18 @@ func DeleteProjectProjectNameStageStageNameServiceServiceNameResourceResourceURI
 	return stage_resource.NewPutProjectProjectNameStageStageNameResourceResourceURICreated().WithPayload(metadata)
 }
 
+func AddUpdateHelmResources(resource string, actionType string, filePath string, projectName string) middleware.Responder {
+	logger.Debugf("%sing resource: %s", actionType, filePath)
+	err := common.WriteBase64EncodedFile(filePath, resource)
+	if err != nil {
+		logger.WithError(err).Errorf("Could not %s resource %s to project %s", actionType, filePath, projectName)
+		return service_resource.NewPostProjectProjectNameStageStageNameServiceServiceNameResourceBadRequest().
```

Comment:
Imho this function should only return an error instead of also generating the complete http response object

Reply:
resolved